tests/integration/test_api_gateway.py

```diff
@@ -127,3 +148,40 @@ def forward_request(self, **kwargs):
 
     # clean up
     proxy.stop()
+
+
+# def test_api_gateway_lambda_proxy_integration():
```

Comment:
If you want me to completely remove it, I can. It's just that I'm working on another PR to fix more AWS spec issues and it would be better if this were in as-is. Mostly because I'm finding my lack of experience with the project to mean that I'm not sure how to quickly implement the approach you suggested e.g. setup Lambda, setup API-Gateway, check S3 for successful execution of Lambda, etc.